ts/BUILD.package

```diff
@@ -0,0 +1 @@
+exports_files(["package"])
```

Comment:
What is the "package" file?

Reply:
each npm package we download and extract through `http_archive` has a single directory at the root called `package`.

Reply:
We want that directory to be exposed so we can copy it via `npm_package`